ui/component/comment/index.js

```diff
@@ -6,33 +6,48 @@ import {
   makeSelectThumbnailForUri,
   makeSelectIsUriResolving,
   selectMyChannelClaims,
+  makeSelectMyChannelPermUrlForName,
```

Comment:
The app crashes here before channels have finished loading
https://github.com/lbryio/lbry-redux/blob/master/src/redux/selectors/claims.js#L670